test/bench/document_test.ts

```diff
@@ -48,16 +49,33 @@ const tests = [
           root.k1 = 'v1';
           root.k2 = { k4: 'v4' };
           root.k3 = ['v5', 'v6'];
-          assert.equal(expected, root.toJSON?.());
+          assert.equal(expected, root.toJSON!());
```

Comment:
I don't think it's necessary to test that it works in the benchmark. So I think it's okay to remove this code.
```suggestion
```